### PR TITLE
RK-10249 - Bugsnag user email

### PR DESCRIPTION
**Pull request by niryarden**
goes with https://github.com/Rookout/frontend/pull/3123

get the userEmail from the frontend, save to store, and add it to bugsnag reports

image

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -78,6 +80,12 @@ export const resolvers = {
     settings: (parent: any, args: { settings: Settings }): Settings => {
       return setSettings(args.settings);
     },
+    userEmail: (parent: any, args: { userEmail: string }): boolean => {
+      if (!store.get(USER_EMAIL_KEY, null)) {
```
Comment (dan505512) [inline src/api.ts:84]: Why not override it if it exists?

Comment (niryarden) [reply src/api.ts:84]: why yes? why to spend resources on it?

Comment (dan505512) [reply src/api.ts:84]: Why spend resources checking? It's the same. Also if someone changes users we might want to change the email

**Review by dan505512 — APPROVED** [review]

**Pull request merged**